### DISCUSSION TITLE
Not using gpu when featuring/matching

### DISCUSSION THREAD

**Discussion by ericpan0513** [colmap/colmap#1718] [Q&A]
Hi, I've got a question when using colmap. Here's my environment:

Ubuntu 18.04
3080ti
colmap 3.8
cuda 11.4

The problem is that when I run colmap, I check nvidia-smi and didn't see any process running on GPU.
<img width="537" alt="截圖 2022-12-08 下午1 59 15" src="https://user-images.githubusercontent.com/67946408/206369204-fedbb9bd-64c9-4986-8673-b29caf2a8e22.png">

I have tried using colmap gui to make sure I have checked the use_gpu box and tried every gpu number, but it still didn't work.
<img width="203" alt="截圖 2022-12-08 下午1 57 00" src="https://user-images.githubusercontent.com/67946408/206368867-059cdb61-5f4f-4163-9096-44bbdd4ac08a.png">

Does anyone has a clue on this situation or how to fix it? 

I am currently using VNC to connect to this machine, will that be the reason why can't use GPU?

Thanks a lot!

**Comment by shubham-monarch** [comment]
Can you elaborate your installation steps?

**Reply by QAQQL** [reply]
> Can you elaborate your installation steps?

Can you look at my problem and help me solve it?
thank you
#2608

**Comment by UmeshLearn2324** [comment]
hi @ericpan0513,

Could you please help me with installation of colmap 3.8 on Ubuntu 18.04, below are specs
> CUDA 11.2, V11.2.67
> gcc (Ubuntu 9.4.0-1ubuntu1~18.04) 9.4.0
> cmake version 3.25.2
> ninja 1.8.2

I am getting below errors
>> ```
tmp/CMakeCUDACompilerId.ptx

 Command-line error: invalid option: --no-version-ident



  1 catastrophic error detected in this compilation.

  Compilation terminated.

  # --error 0x1 --
```
`
